app/src/main/java/org/mozilla/focus/telemetry/TelemetrySettingsProvider.java

```diff
@@ -50,7 +50,7 @@ public boolean containsKey(String key) {
             // The default browser is not actually a setting. We determine if we are the
             // default and then inject this into telemetry.
             final Context context = TelemetryHolder.get().getConfiguration().getContext();
-            final Browsers browsers = new Browsers(context, Browsers.TRADITIONAL_BROWSER_URL);
+            final Browsers browsers = new Browsers(context, Browsers.TRADITIONAL_BROWSER_URL, false /*doesn't matter here*/);
```

Comment:
I arbitrary set the `isCustomTab` to `false`, but whether it's `false` or `true` `Browsers.isDefaultBrowser` return the same value.